src/__internal__/validations/validation-icon.style.js

```diff
@@ -1,18 +1,24 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { margin } from "styled-system";
 import BaseTheme from "../../style/themes/base";
 import StyledIcon from "../../components/icon/icon.style";
 
 const ValidationIconStyle = styled.span`
   background: none;
-  cursor: pointer;
+  cursor: default;
   display: flex;
   align-items: center;
   margin-top: 0;
   margin-bottom: 0;
   border: 0;
   outline: none;
 
+  ${({ isPartOfInput }) =>
```

Comment:
```
Add conditional styling to ValidationIcon such that, on hover, the cursor becomes a pointer only if the icon is not part of some input. This is to avoid confusion that an action is to be followed after clicking the icon.
```

This will add a pointer cursor unless I'm misunderstanding your intention here

Reply:
Thanks for your comment @edleeks87. Changing the cursor to a pointer was intentional here, so for other components such as `DateRange` and `DateInput`, where the icon forms part of the input element, the pointer does not disappear. But for all other cases like `RadioButton`, the default cursor will be shown, as clicking on the validation icons doesn't trigger an action.

Reply:
No problem, I thought it might have been the PR description that was off tbh